.gitmodules

```diff
@@ -15,4 +15,4 @@
 [submodule "submodules/xdp-for-windows"]
 	path = submodules/xdp-for-windows
 	url = https://github.com/microsoft/xdp-for-windows.git
-	branch = release/1.0
+	branch = 300d18c43f989ca22ad1be2f2836e2bbec6a2c4f
```

Comment:
@mtfriesen what are your thoughts on a timeline for a new release branch (not necessarily an official release) from XDP?

Reply:
I'm not entirely sure I understand the distinction - XDP should have a release branch 1:1 with each major/minor version. Are you asking if we can do an unsupported/unsigned release for 1.x, and then either make it "official" later, or let 1.y become officially supported afterwards?

Reply:
I'm not asking for a one-off. I'm asking: when should we fork the next XDP release branch?

Reply:
Whenever needed by a dependent project. XDP keeps `main` in a releasable state at all times.